Services/DataBaseService/Repositories/UserRepository.cs

```diff
@@ -38,16 +38,16 @@ public UserCredential GetUserCredential(string email)
 
             return mapper.MapUserCredential(dbCredential);
         }
-        public void DeleteUser(Guid userIdCredential)
+        public void DeleteUser(Guid userId)
         {
-            var dbUserCredential = dbContext.Find<DbUserCredential>(userIdCredential);
+            var dbUserCredential = dbContext.UsersCredentials.FirstOrDefault(uc => uc.UserId == userId);
             if (dbUserCredential == null)
             {
                 throw new ForbiddenException("Not found User for delete");
             }
             if(dbUserCredential.IsActive==false)
             {
-                throw new ForbiddenException("User was deleted early");
+                throw new ForbiddenException("User was deleted early or not confirmed");

```

Comment:
I still don't really understand "or not comfirmed part" to be honest

Reply:
In order for the user to have an IsActive field other than 0, they must confirm themselves using email.
And in this case, If IsActive is 0, it`s mean that user was deleted early or was not confirmed)

Reply:
> In order for the user to have an IsActive field other than 0, they must confirm themselves using email.
> And in this case, If IsActive is 0, it`s mean that user was deleted early or was not confirmed)

Ah, I got it know, thanks =)